params/governance_params.go

```diff
@@ -104,6 +105,7 @@ var (
 	DefaultStakeUpdateInterval       = uint64(86400) // 1 day
 	DefaultProposerRefreshInterval   = uint64(3600)  // 1 hour
 	DefaultPeriod                    = uint64(1)
+	DefaultDeriveShaImpl             = uint64(0) // Orig
```

Comment:
I cannot because that would create an import cycle.
```
package github.com/klaytn/klaytn/params
        imports github.com/klaytn/klaytn/blockchain/types
        imports github.com/klaytn/klaytn/blockchain/types/accountkey
        imports github.com/klaytn/klaytn/fork
        imports github.com/klaytn/klaytn/params: import cycle not allowed
```

I did once moved `types.ImplDeriveSha*` enums to `params.DeriveShaImpl_*` as you have seen.
Later, I did undo that because it seemed irrelevant to the current change.

Reply:
Let's move to [here](https://github.com/klaytn/klaytn/pull/1708#pullrequestreview-1189105436)